include/bluetooth/mesh/main.h

```diff
@@ -93,6 +120,21 @@ struct bt_mesh_prov {
 	/** Supported Input OOB Actions */
 	uint16_t       input_actions;
 
+	/** @brief Provisioning Capabilities.
+	 *
+	 *  This callback notifies the application that the provisioning capabilities
+	 *  of the unprovisioned device has been received.
+	 *
+	 *  The application can consequently call bt_mesh_auth_method_set_<*> to
+	 *  select suitable provisioning oob authentication method.
+	 *
+	 *  If this callback set to NULL, no authentication will be used when
+	 *  provisioning other devices.
```

Comment:
This sentence is no longer true.
```suggestion
	 *  When this callback returns, the provisioner will start 
	 *  authentication with the chosen method.
```